tests/test_options.py

```diff
@@ -8,6 +8,15 @@
 from sphinx.util.images import get_image_size
 from sphinxcontrib.drawio import DrawIOError
 
+def image_size_equal(a, b, blur: int = 2) -> bool:
```

Comment:
It's because I can't get the test page-name related test case passed.

Reply:
> Sorry, what's the point of this?
> 
> Also - please don't make unrelated test changes within this PR; it should be separate

Should the page-name related test case be a separate PR too?

Reply:
> No, the test cases should be together with the feature addition.
> 
> If you're talking about the image sizes being one or two pixels off - yes, that happens with different draw.io versions. I haven't bumped the version of draw.io in CI in a while now, so just working on doing that + fixing the different image size tests (and fixing the fact a python version isn't deployed by github anymore) and that should fix that.
> 
> I'm also going to remove the deprecated figure directives since it's been a few years and that should be long enough now.

Ok, it's fine, this case failed too.
 

======================================================== test session starts =========================================================platform darwin -- Python 3.9.10, pytest-7.2.0, pluggy-1.0.0
rootdir: /Volumes/work/src/sphinxcontrib-drawio
plugins: Faker-8.8.1
collected 25 items                                                                                                                   

tests/test_caching.py .                                                                                                        [  4%]
tests/test_compat.py s                                                                                                         [  8%]
tests/test_options.py .........F.....                                                                                          [ 68%]
tests/test_pdf.py ....                                                                                                         [ 84%]
tests/test_update.py ....                                                                                                      [100%]

============================================================== FAILURES ==============================================================____________________________________________________________ test_figure _____________________________________________________________
content = <SphinxTestApp buildername='html'>
directives = [<img alt="_images/box.png" class="drawio" src="_images/box.png"/>, <img alt="_images/box1.png" class="drawio" src="_images/box1.png"/>]

    @pytest.mark.sphinx("html", testroot="figure")
    def test_figure(content: Sphinx, directives: List[Tag]):
        filenames_sizes = [
            ("box.png", (125, 64)),
            ("box1.png", (185, 94)),
        ]
        for img, (filename, size) in zip(directives, filenames_sizes):
            assert img.name == "img"
            assert img["src"] == "_images/" + filename
            assert img["alt"] == "_images/" + filename
            assert img["class"] == ["drawio"]
            image_path = content.outdir / img["src"]
            assert image_size_equal(get_image_size(image_path), size)
            imageContainerTag = img.parent
>           assert imageContainerTag.name == "figure"
E           AssertionError: assert 'div' == 'figure'
E             - figure
E             + div

tests/test_options.py:141: AssertionError

Reply:
> 

I think we need to add this fuzzy value to be compatible with multiple versions of the draw.io program.

Reply:
> Possibly - if anything those tests shouldn't be checking against image size, it's just a dirty way to verify an image was actually produced. For the tests which specify an exact image size draw.io does produce consistent output sizes.
> 
> If you can remove this from this PR at least that'd be great - I'm open to discussing adding it in another PR or figuring out how to make upstream produce stable sizes.

Ok, I removed it, will submit another PR after this change is merged.